pubsub/envelope.go

```diff
@@ -24,6 +24,7 @@ const (
 	DefaultCloudEventSource = "Dapr"
 	// DefaultCloudEventDataContentType is the default content-type for the data attribute
 	DefaultCloudEventDataContentType = "text/plain"
+	DaprTraceIDField                 = "daprtraceid"
```

Comment:
We have chosen not to name `expiration` as `daprexpiration`. Why is this field different?

Reply:
I'll change it to match

Reply:
Which field will change?